src/common/KakaoLogin.tsx

```diff
@@ -0,0 +1,13 @@
+import Link from "next/link"
+
+import { KAKAO_AUTH_URL } from "@/constants"
+
+const KakaoLogin = () => {
+  return (
+    <Link style={{ backgroundColor: "yellow" }} href={KAKAO_AUTH_URL}>
+      카카오톡 로그인
+    </Link>
```

Comment:
css를 적용할 때 tailwind가 아닌 style을 사용한 이유가 있을까요?
정식으로 버튼을 만들땐 kakao 색상을 사용하면 좋을 것 같아요.

```suggestion
    <Link className="bg-kakao" href={KAKAO_AUTH_URL}>
      카카오톡 로그인
    </Link>
```

Reply:
앗 아뇨 테스트용이라서 저렇게 해뒀는데 스타일링할때 수정하겠습니다! 감사합니다!